boards/common/cc2538/Makefile.include

```diff
@@ -1,9 +1,6 @@
 # define the default flash-tool
 PROGRAMMER ?= cc2538-bsl
 
-# setup serial terminal
```

Comment:
The serial port is also used for the `BSL` here, it needs to be kept somehow.

Reply:
I moved it to the cc2538-bsl programmer code